src/core/include/openvino/core/rtti.hpp

```diff
@@ -24,8 +24,18 @@
 #define _OPENVINO_RTTI_WITH_TYPE_VERSION_PARENT(TYPE_NAME, VERSION_NAME, PARENT_CLASS) \
     _OPENVINO_RTTI_WITH_TYPE_VERSIONS_PARENT(TYPE_NAME, VERSION_NAME, PARENT_CLASS, 0)
 
+#if defined _WIN32 || defined __CYGWIN__
+#    define _HIDDEN
+#else
+#    if __GNUC__ >= 4
+#        define _HIDDEN __attribute__((visibility("hidden")))
+#    else
+#        define _HIDDEN
+#    endif
+#endif
+
 #define _OPENVINO_RTTI_WITH_TYPE_VERSIONS_PARENT(TYPE_NAME, VERSION_NAME, PARENT_CLASS, OLD_VERSION) \
-    static const ::ov::DiscreteTypeInfo& get_type_info_static() {                                    \
+    _HIDDEN static const ::ov::DiscreteTypeInfo& get_type_info_static() {                            \
```

Comment:
Please do the same thing in the line 15.

Reply:
Done, thanks!